### PR TITLE
Feature/group creation

### PR DESCRIPTION
**Pull request by Puh00**
Implemented group creation functionality.

### CODE REVIEW

**Comment by yenanw** [comment]
closes #84

File: app/src/main/java/com/goayo/debtify/view/CreateGroupFragment.java [hunk 1]
```diff
@@ -4,36 +4,106 @@
...
 
 /**
  * @author Alex Phu
  * @date 2020-09-16
  * <p>
  * Create-a-group page.
- *
+ * <p>
  * 2020-09-18 Modified by Alex Phu and Olof Sjögren: Added listener for create button. Logic will be implemented later.
+ * 2020-10-05 Modified by ALex & Yenan: implemented
```
Comment (OlofSjogren) [inline app/src/main/java/com/goayo/debtify/view/CreateGroupFragment.java:37]: Implemented what?

File: app/src/main/java/com/goayo/debtify/view/CreateGroupFragment.java [hunk 2]
```diff
@@ -4,36 +4,106 @@
...
+        NavHostFragment.findNavController(this).navigate(R.id.action_createGroupFragment_to_pickUsersFragment);
+    }
+
+    private void createGroup() {
+        String groupName = binding.createGroupGroupNameTextView.getText().toString();
+        List<IUserData> userData = userCardViewAdapter.getUserList();
+
+        try {
+            if (groupName.isEmpty()) {
+                throw new Exception("YOUR FUCKING NAME IS EMPTY");
```
Comment (OlofSjogren) [inline app/src/main/java/com/goayo/debtify/view/CreateGroupFragment.java:94]: Watch your profanity!

**Review by OlofSjogren — APPROVED** [review]

File: app/src/main/java/com/goayo/debtify/view/CreateGroupFragment.java (hunk 2 above)
Comment (OscarSanner) [inline app/src/main/java/com/goayo/debtify/view/CreateGroupFragment.java]: This won't be shown to the users right? Inapropes

**Review by OscarSanner — APPROVED** [review]

**Pull request merged**